Spikes/movement/Character.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifndef CHARACTER_H
+#define CHARACTER_H
+
+#include "SFML/Graphics.hpp" 
+#include "SFML/OpenGL.hpp" 
+
+using namespace sf;
+
+
+
+class Character
+{
+private:
+	//Character
+	bool m_move;
+	Vector2f m_centre;
+	Vector2f m_pos;
+	float m_radius;
+	float m_speed;
+	float m_angle;
+	CircleShape m_shape;
+
+	//Target
+	Vector2f m_targetPos; 
+	float m_targetRadius;
+	float m_targetAngle;
+	CircleShape m_target;
+
+public:
+	Character();
+	Character(float, float, Color, Color, Vector2f);
+	void update();
+	void draw(RenderWindow&);
+	void resetAngles();
+	float findAngleBetween();
+
+	// Get methods
```

Comment:
Note: findAngleBetween() is not a method of current Vector2D class so will be left alone until updated